core/azure-core/src/main/java/com/azure/core/polling/PollerOptions.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+
+/**
+ * This class provide configuration options needed to create Poller.
+ *
+ * <p><strong>Place holder for Poller Options</strong></p>
+ */
+public class PollerOptions {
```

Comment:
If you can't think of any additional options we may want to expose, it might be best to get rid of PollerOptions and simply add a `Duration pollInterval` API to the Poller API instead.

Reply:
We do not need to change our API when we need additional values in Poller Options. It gives us little bit of future proof.

Reply:
Yes. If there are further options we may want to support, that is the argument to make. I'm just asking if you can foresee any other options we may need to expose, or if this is the totality of it.

Reply:
I do not see additional config parameter need for now based on spec. We could get rid of PollerOptions for now.  I would remove it, no need to over engineer at this point. You are okay with this decision ?

Reply:
Yes

Reply:
done